src/store/getters/state-getters.ts

```diff
@@ -746,11 +746,14 @@ export const havePeopleAndRolesChanged = (state: StateIF): boolean => {
       if (!op.deliveryAddress.deliveryInstructions) op.deliveryAddress.deliveryInstructions = ''
       if (!op.deliveryAddress.streetAddressAdditional) op.deliveryAddress.streetAddressAdditional = ''
     }
+
     if (op.mailingAddress) {
       if (!op.mailingAddress.deliveryInstructions) op.mailingAddress.deliveryInstructions = ''
       if (!op.mailingAddress.streetAddressAdditional) op.mailingAddress.streetAddressAdditional = ''
     }
+
     if (!op.officer.email) op.officer.email = null
+    if (!op.officer.taxId) op.officer.taxId = null
```

Comment:
Some officers have taxId=null (or a value) and some objects don't. This code is needed so the comparison below doesn't have false positives.

This was causing a change/conversion filing to show a change (and allow the user to go to the Review page) even when nothing had changed.